sbi/utils/dkl.py

```diff
@@ -0,0 +1,43 @@
+import torch
+import torch.distributions
+from sbi.inference.posteriors.sbi_posterior import Posterior
+from typing import Union
+
+
+def dkl_monte_carlo_estimate(
```

Comment:
maybe call it `get_dkl_monte_carlo_estimate` or `get_dkl_via_monte_carlo`?

Reply:
I personally am not so much in favor of `get_`. You shall `get` from anything that `Return`s ;)

Reply:
dkl_via_monte_carlo it is ;)

Reply:
OK, now Michael you can decide :)